### PR TITLE
feature: Stabilize Compute Costs

### PR DESCRIPTION
**Pull request by akashin**
Remove the compile time feature "protocol_feature_compute_costs" and replace all checks in the code to look at the current protocol version instead.

### CODE REVIEW

File: runtime/runtime/src/lib.rs [hunk 1]
```diff
@@ -2562,8 +2558,18 @@ mod tests {
...
+            compute: if checked_feature!(
+                "stable",
+                ComputeCosts,
+                apply_state.current_protocol_version
+            ) {
+                Compute::from(10_000_000_000_000u64)
+            } else {
+                Compute::from(1_000_000u64)
+            },
+        };
```
Comment (jakmeier) [inline runtime/runtime/src/lib.rs:2572]: Unless you want to add a test that checks a version before the feature was added, I don't think you need this `checked_feature!`, right? (Assuming we bump the protocol version first)

File: core/primitives/src/version.rs [hunk 2]
```diff
@@ -238,6 +242,7 @@ impl ProtocolFeature {
             ProtocolFeature::Ed25519Verify
             | ProtocolFeature::ZeroBalanceAccount
             | ProtocolFeature::DelegateAction => 59,
+            ProtocolFeature::ComputeCosts => 61,
```
Comment (jakmeier) [inline core/primitives/src/version.rs:245]: When (or before) we can put stable protocol features to version 61, we have to bump `STABLE_PROTOCOL_VERSION` to 61. We usually do it in the same PR as the feature stabilization, so you can do it directly in this PR. That should also make stable tests easier.

**Review by jakmeier — CHANGES_REQUESTED** [review]

**Comment by akashin** [comment]
I've replaced this PR with https://github.com/near/nearcore/pull/8915

File: core/primitives/src/version.rs (hunk 2 above)
Comment (akashin) [reply core/primitives/src/version.rs:245]: I've addressed this in https://github.com/near/nearcore/pull/8915

File: runtime/runtime/src/lib.rs (hunk 1 above)
Comment (akashin) [reply runtime/runtime/src/lib.rs:2572]: I've addressed this in https://github.com/near/nearcore/pull/8915

**Pull request closed without merging**